test/int64.js

```diff
@@ -0,0 +1,663 @@
+    },
+    {
+      value: Number.MAX_SAFE_INTEGER,
+      expectedString: Number.MAX_SAFE_INTEGER.toString(),
+      message: 'must create Int64 object from a big number',
+    },
+    {
+      value: -Number.MAX_SAFE_INTEGER,
+      expectedString: '-' + Number.MAX_SAFE_INTEGER.toString(),
+      message: 'must create Int64 object from a big number',
+    },
+    {
+      value: {},
+      expectedString: '0',
+      message: 'must create zero Int64 from an invalid constructor argument',
+    },
+    {
+      value: NaN,
+      expectedString: '0',
+      message: 'must create zero Int64 from a NaN constructor argument',
+    },
+    {
+      value: undefined,
+      expectedString: '0',
+      message: 'must create zero Int64 from an undefined constructor argument',
+    },
+    {
+      value: '0xinvalidnumber',
+      expectedString: '0',
+      message:
+        'must create zero Int64 from an invalid hex string',
+    },
+    {
+      value: '0o999',
+      expectedString: '0',
+      message:
+        'must create zero Int64 from an invalid oct string',
+    },
+    {
+      value: '0b333',
+      expectedString: '0',
+      message:
+        'must create zero Int64 from an invalid bin string',
+    },
+    {
+      value: 9999999999999999999999999999999999,
+      expectedString: '0',
+      message:
+        'must create zero Int64 from an unsafe integer number',
+    },
+    {
+      value: -9999999999999999999999999999999999,
+      expectedString: '0',
+      message:
+        'must create zero Int64 from an unsafe integer number',
+    },
+    {
+      value: '0xffffffffffffffff',
+      expectedString: '-1',
+      message: 'must create Int64 from a big hex number provided as a string',
+    },
+    {
+      value: '0x7fffffffffffffff',
+      expectedString: '9223372036854775807',
+      message: 'must create Int64 from a big hex number provided as a string',
+    },
+    {
+      value:
+        '0b1010101010101010101010101010101010101010101010101010101010101010',
+      expectedString: '-6148914691236517206',
+      message: 'must create Int64 from a big bin number provided as a string',
+    },
+    {
+      value:
+        '0b10101010101010101010101010101010101010101010101010101010101010',
+      expectedString: '3074457345618258602',
+      message: 'must create Int64 from a big bin number provided as a string',
+    },
+    {
+      value: '0o1777777777777777777777',
+      expectedString: '-1',
+      message: 'must create Int64 from a big oct number provided as a string',
+    },
+    {
+      value: '0o777777777777777777777',
+      expectedString: '9223372036854775807',
+      message: 'must create Int64 from a big oct number provided as a string',
+    },
+    {
+      value: '9223372036854775807',
+      expectedString: '9223372036854775807',
+      message: 'must create Int64 from a big dec number provided as a string',
+    },
+    {
+      value: '-9223372036854775807',
```

Comment:
Why not use `-9223372036854775808` here as the minimal int64 value, similar to the test case above, where you use the maximal int64 value?